js/src/Ice/WSTransceiver.js

```diff
@@ -25,11 +25,26 @@ if (typeof WebSocket !== "undefined") {
     const StateClosed = 4;
 
     WSTransceiver = class {
-        constructor(instance) {
+        constructor(instance, secure, addr, resource) {
```

Comment:
The transceiver construction logic was split between the constructor and a static method. That is due to how we previously include the classes in NodeJS and Browser. With the current code we can use the constructors directly witch is clear.